custom/icds_reports/templates/icds_reports/icds_app/cas-export.directive.html

```diff
@@ -4,7 +4,8 @@
             <span class="breadcrump-element">Export CAS Data: </span>
         </div>
     </div>
-    <info-message></info-message>
+    <info-message start="1" end="6" type="download-7"></info-message>
+    <info-message start="7" end="31" type="download-7-last"></info-message>
```

Comment:
based on the name should this not be `start="24"` or something? (caveat: I have no idea how these work)

Reply:
looked at the spec and think the logic is right. but maybe the "type" is oddly named? what were you thinking `download-7-last` meant?

Reply:
I've updated the type and made them more verbose, do let me know if you have any thoughts on it.

Reply:
looks good, thanks